README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_gallery_DOM/)
+    - [DEMO LINK](https://vladyslava-buzova.github.io/js_gallery_DOM/)
```

Comment:
add to PR's description and deploy

Reply:
[DEMO LINK](https://vladyslava-buzova.github.io/js_gallery_DOM/)